.github/workflows/lint.yml

```diff
@@ -44,3 +44,8 @@ jobs:
       run: |
         pip install black
         black --check .
+
+    - name: Type Checking with MyPy
+      run:  |
+        pip install mypy
+        mypy
```

Comment:
Lets make this || true for now. Mypy is best as a warning not an error in its current state